concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/80 Export Data.md

```diff
@@ -0,0 +1,289 @@
+DataGrid allows users to export grid data as Excel and PDF documents. This feature requires the following third-party libraries:
+
+- Excel: <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v4+ and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> v2.0.2+
+
+- PDF: <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a>
+
+To configure export operations, use the **excelExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) or **pdfExporter**.[exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method. It requires a DataGrid instance and a target Excel worksheet or a PDF document. You can call these methods at any point in your application.
+
+The following code calls export methods in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The control executes this handler when users select one of the operations available via the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#formats) property to configure available export operation types.
```

Comment:
```suggestion
The following code calls export methods in the DataGrid's [onExporting](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onExporting) event handler. The control executes this handler when users select one of the operations available using the **Export** button. The [export](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/).[enabled](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#enabled) property adds this button to the DataGrid. Use the [formats](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/export/#formats) property to configure available export operation types.
```

Reply:
I decided to rephrase this sentence since I don't want to use 'using' in the sentences.
What do you think about the following?

> The control executes this handler when users select one of the operations available in the **Export** button drop-down menu.

Reply:
this works